lib/ContactsMenu/Providers/ChatProvider.php

```diff
@@ -52,7 +53,7 @@ public function process(IEntry $entry)
 			$config = \OC::$server->getConfig();
 			$xmppPreferMail = $config->getAppValue('ojsxc', 'xmppPreferMail', false);
 			$xmppPreferMail = $xmppPreferMail === true || $xmppPreferMail === 'true';
-			$serverType = $config->getAppValue('ojsxc', 'serverType', 'external');
```

Comment:
@sualko This was not intentional I guess? (That the default is external)

Reply:
You are right :+1:

Reply:
btw I added another commit which substitutes all server type string literals with constants.